Directory.Build.props

```diff
@@ -16,6 +16,8 @@
         <EnableNETAnalyzers>true</EnableNETAnalyzers>
         <AnalysisLevel>latest</AnalysisLevel>
         <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
+
+        <NoWarn>$(NoWarn);NCRONJOB_REPORTING</NoWarn>
```

Comment:
I haven't been able to successfully move that silencing to `.editorconfig` :-/

cf. https://learn.microsoft.com/en-us/dotnet/fundamentals/syslib-diagnostics/experimental-overview#suppress-warnings